build.sbt

```diff
@@ -1503,9 +1504,10 @@ ThisBuild / dependencyOverrides ++= Seq(
   "io.netty" % "netty-tcnative-boringssl-static" % nettyTcNativeVersion,
   "io.netty" % "netty-transport" % nettyVersion,
   "io.opencensus" % "opencensus-api" % opencensusVersion,
+  "io.opencensus" % "opencensus-contrib-grpc-metrics" % opencensusVersion,
   "io.opencensus" % "opencensus-contrib-grpc-util" % opencensusVersion,
   "io.opencensus" % "opencensus-contrib-http-util" % opencensusVersion,
-  "io.opencensus" % "opencensus-contrib-grpc-metrics" % opencensusVersion,
+  "io.perfmark" % "perfmark-api" % perfmarkVersion,
```

Comment:
just wondering, what's this dependency needed for?

Reply:
The lib creates a warning in `scio-google-cloud-platform/evicted`